test/examples_test.go

```diff
@@ -171,3 +222,31 @@ func TestTWAPOnChain(t *testing.T) {
 		NPMInstall().
 		Run("--grep", "Local")
 }
+
+func TestTWAPCombined(t *testing.T) {
```

Comment:
is the "combined" name fine or is there a better way to describe these combination tests?

Reply:
seems fine to me. Could use "compatibility" like in zpr or you could name it something like TestTWAPExampleDependencies or TestTWAPIOCoupledExamples

Reply:
dependencies is kind of an overloaded term though

Reply:
Im just going to leave combined for now. Works well enough and I don't want to get held up on naming